vertica/datadog_checks/vertica/vertica.py

```diff
@@ -92,6 +92,46 @@ def __init__(self, name, init_config, instances):
         # Cache database results for re-use among disparate functions
         self._view = defaultdict(list)
 
+        self._metric_groups = {
```

Comment:
Since we still support Python 2 for now, we'll need to guarantee order via `OrderedDict`, see https://github.com/DataDog/integrations-core/blob/1fa5db498c3d08edf88f3e83c14f227724d1d320/datadog_checks_base/datadog_checks/base/stubs/aggregator.py#L50-L60